client/css/main.scss

```diff
@@ -103,3 +103,58 @@ md-input-container {
   background-color: #68b7fc;
 }
 
+.p { 
```

Comment:
This class name is a bit confusing, at first I thought it was styling every paragraph.